src/main/java/dev/jbang/cli/BaseBuildCommand.java

```diff
@@ -76,14 +34,6 @@ void setJavaVersion(String javaVersion) {
 		this.javaVersion = javaVersion;
 	}
 
-	@CommandLine.Option(names = {
```

Comment:
`--cds` seem to have been removed fully for build that does not seem right?

Reply:
Well unless I'm misunderstanding the docs, the -X options for CDS only exist for the `java` command, not `javac`? So that would mean it's useless for the `jbang build` command AFAICS.

Reply:
Also, the `cds` value was only referenced from the `Run` class which seems to corroborate that it's not for build-time.

Reply:
you are right.